### ISSUE TITLE
version mismatch with wrenc?

### ISSUE THREAD

**Issue by colinleach** [exercism/wren#124]
Your instructions for installing wren locally on Linux (https://exercism.org/docs/tracks/wren/installation) are for wrenc v0.2.90 (wren v0.4.0). This installed successfully and I could run wrenc to get a console prompt, but package installation failed. Running `wrenc package.wren install` gave the error "Could not load module 'wren-package'." and I got a similar error from the wren console prompt.

The repo https://github.com/joshgoebel/wren-console/releases currrently gives v0.3.1 (dated Nov 2021), and this seems to have none of the same problems: at least with hello-world, I haven't got further yet. I'm not sure of the best way to respond - maybe update the installation instructions to v0.3??

Thanks for providing this resource, I now look forward to learning the language.

**Comment by joshgoebel** [comment]
Exactly. I think I completed the docs before everything was good and ready. I'll get the version numbers updated soon. Thanks.

**Comment by joshgoebel** [comment]
@colinleach Check #125

**Comment by colinleach** [comment]
I'm a bit confused by the github workflow, but the changes look good - thank you!

**Comment by joshgoebel** [comment]
New version of libc incompatible with Wren 0.4.0... it needs a small patch, but that's not going to matter for just a docfix.

**Comment by colinleach** [comment]
libc is far beyond my expertise (sorry - I'm a retired organic chemist, not an IT professional). What I will say is that I came to Wren via the 12in23 challenge and I very much hope others will too. It would be a pity to repel them at Hello World: your language deserves more respect.

**Comment by joshgoebel** [comment]
Not my language. I only helped build out the track and tooling. But yes, Wren is awesome.

**Comment by joshgoebel** [comment]
The docs say 3.1 now... can this be closed?
